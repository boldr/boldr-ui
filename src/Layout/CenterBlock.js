/* @flow */
import * as React from 'react';

import Row from './Row';
import Col from './Col';

type Props = {
  children: Array<React.Node>,
};

const CenterBlock = (props: Props) => {
  return (
    <Row xsCenter>
      <Col xs={6}>{props.children}</Col>
    </Row>
  );
};

export default CenterBlock;
