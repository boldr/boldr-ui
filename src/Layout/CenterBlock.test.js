import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import CenterBlock from './CenterBlock';
//
describe('<CenterBlock />', () => {
  test('<CenterBlock />, should match the snapshot', () => {
    const wrapper = shallow(<CenterBlock />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

  test('It should render with correct classes', () => {
    const wrapper = mount(<CenterBlock />);
    const wrapperInner = wrapper.find('.grid__row');
    expect(wrapperInner.is('.grid__row')).toBe(true);
  });
});
