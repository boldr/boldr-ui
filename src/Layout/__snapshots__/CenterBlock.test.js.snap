// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CenterBlock /> <CenterBlock />, should match the snapshot 1`] = `
<Row
  componentClass="div"
  xsCenter={true}
>
  <Col
    componentClass="div"
    xs={6}
  />
</Row>
`;
